input/data/2009/2009ApJ...690L.126A/tev-000036-sed.ecsv

```diff
@@ -5,16 +5,20 @@
 # - {name: dnde, unit: cm-2 s-1 TeV-1, datatype: float32}
 # - {name: dnde_errn, unit: cm-2 s-1 TeV-1, datatype: float32}
 # - {name: dnde_errp, unit: cm-2 s-1 TeV-1, datatype: float32}
+# - {name: dnde_ul, unit: cm-2 s-1 TeV-1, datatype: float32}
 # meta: !!omap
 # - {data_type: sed}
 # - {source_id: 36}
 # - {reference_id: 2009ApJ...690L.126A}
 # - {telescope: veritas}
 # - mjd: {max: 54586.0, min: 54040.0}
+# - UL_CONF: 0.90
 # - {comments: This data was collected for 2015ApJ...812...60B and contributed to gamma-cat
 #     by Jonathan Biteau.}
```

Comment:
Since the data now no longer matches the one from 2015ApJ...812...60B, can you please update this comment?
Either add a short line like "Data from ???" (URL or corresponding author internal note or data picker or whatever), or just replace the existing line, since now it's no longer important that an earlier version came from the 2015ApJ...812...60B data collection.

Reply:
Changed the comment to make clear that the UL is from the original authors.